--- a/www/manager6/node/StatusView.js
+++ b/www/manager6/node/StatusView.js
@@ -110,7 +110,13 @@
                     return data.kversion;
                 }
                 let kernel = data['current-kernel'];
-                let buildDate = kernel.version.match(/\((.+)\)\s*$/)?.[1] ?? 'unknown';
+                let buildDate;
+                try {
+                    buildDate = kernel.version.match(/\((.+)\)\s*$/)[1] ?? 'unknown';
+                } catch {
+                // Patch for some kernels not match (date).
+                    buildDate = kernel.version ?? 'unknown';
+                }
                 return `${kernel.sysname} ${kernel.release} (${buildDate})`;
             },
             value: '',
